Introduce LF ending for some popular file types

foxtran opened the pull request.

alexvakimov commented:
Thank you Igor @foxtran  for a helpful idea, but this change affects too many files in by removing and adding them (trivial change). Could you please edit your PR to show only the actual changes?

foxtran commented:
@alexvakimov, I split my commit into two: in the first one, I added `.gitattributes` file, then, in the second I updated all affected files. Hope, it will help :-)
